spec/factories.rb

```diff
@@ -14,35 +14,35 @@ def r_str
 
 require "diaspora_federation/test/factories"
 
-FactoryGirl.define do
+FactoryBot.define do
   factory :profile do
     sequence(:first_name) { |n| "Robert#{n}#{r_str}" }
     sequence(:last_name)  { |n| "Grimm#{n}#{r_str}" }
-    bio "I am a cat lover and I love to run"
-    gender "robot"
-    location "Earth"
-    birthday Date.today
-    tag_string "#one #two"
+    bio { "I am a cat lover and I love to run" }
+    gender { "robot" }
+    location { "Earth" }
+    birthday { Date.today }
```

Comment:
Rails/Date: Do not use `Date.today` without zone. Use `Time.zone.today` instead.